### ISSUE TITLE
Would it be possible to add keyboard navigation?

### ISSUE THREAD

**Issue by magnuslsjoberg** [CDrummond/lms-material#478]
With keyboard navigation it would be possible to run the web UI on a TV and use a remote control to navigate.
Perhaps a special layout "layout=tv" could be designed.

For examples of keyboard navigation see for example: [https://webaim.org/techniques/keyboard/](https://webaim.org/techniques/keyboard/)

**Comment by CDrummond** [comment]
Sorry, but not without a **lot** of work. I looked into this briefly as it was mentioned on the LMS forums. I managed to get keyboard navigation of the browse view working (up, down, enter, etc). The problem is when I show a menu (e.g. when pressing 'enter' on a audio item) that does not seem to get keyboard focus, so up/down did not work there. I'd also 

Material has just not been designed for this, and it is a use case I just do not have. Therefore, I'm not going to spend time on this.
